### ISSUE TITLE
[enhancement] User customize button image

### ISSUE THREAD

**Issue by haniunc** [TheInformationLab/Tableau-Extension-ExportAll#8]
First of, the new version functionality is great!

For an enhancement, is it possible to add an option where the user can upload the button image?

Thanks!

**Comment by craigbloodworth** [comment]
Great idea. Just to clarify you want the image to replace the standard button in use now?

Where would you want to store the image? Would you want me to store it for you or would you want to host it on your own url? Or have the option of either?

**Comment by haniunc** [comment]
Yes, it would replace the standard button, if possible. The reason for this is that with the past versions, I used the icon only without text to indicate that the data can be exported. It was small and easy to place anywhere on a dashboard. 

As for storage, either option would work. Would it be possible to have it embedded in the dashboard file, similar to how the native Tableau button functions now?

Thanks again for this awesome extension! It's been extremely helpful

**Comment by mayurdesai24** [comment]
@craigbloodworth me too expecting this one. small excel icon is also fine
